src/app/components/navbar/navbar.component.ts

```diff
@@ -41,7 +44,17 @@ export class NavbarComponent implements OnInit {
       }, 1000);
       return false;
     } else {*/
-      this.onOpenModalEvent();
+    this.onOpenModalEvent();
     //}
+
+  }
```

Comment:
This files changes were my attempt to build a simpler nav bar without flexbox - I ended up running into the same mobile disappear issue which makes me think something else is happening.